next/13/replace-next-head/test.ts

```diff
@@ -47,19 +48,94 @@ const transform = async (json: DirectoryJSON) => {
 		parseMdx: typeof parseMdx;
 		stringifyMdx: typeof stringifyMdx;
 		visitMdxAst: typeof visit;
+		filterMdxAst: typeof filter;
 		unifiedFileSystem: UnifiedFileSystem;
 	}>(unifiedFileSystem, () => ({
 		tsmorph,
 		parseMdx,
 		stringifyMdx,
 		visitMdxAst: visit,
+		filterMdxAst: filter,
 		unifiedFileSystem,
 	}));
 
 	return executeRepomod(api, repomod, '/', {});
 };
 
 describe('next 13 replace-next-head', function () {
+	afterEach(() => {
```

Comment:
I'd say we should ensure there is nothing set before we run a test, so I'd use `beforeEach` here